repo2docker/app.py

```diff
@@ -122,6 +122,17 @@ def fetch(self, url, ref, checkout_path):
                                extra=dict(phase='failed'))
                 sys.exit(1)
 
+    def validate_image_name(self, image_name):
+        """
+        Validate that the image_name read by argparse contains only lowercase characters
+        :param image_name: string argument read by argparse
```

Comment:
Could you use either [Google-style docstrings](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) or [numpydoc docstrings](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#docstring-standard)?

Reply:
Fixed